src/components/Common/UserAutocompleteFormField.tsx

```diff
@@ -80,8 +80,11 @@ export default function UserAutocomplete(props: UserSearchProps) {
         )}
         optionLabel={formatName}
         optionIcon={userOnlineDot}
+        optionImage={(option: UserBareMinimum) =>
+          option.read_profile_picture_url || ""
+        }
```

Comment:
Like already said in previous review: https://github.com/ohcnetwork/care_fe/pull/8903#discussion_r1813659763

```suggestion
        optionImage={(option: UserBareMinimum) => option.read_profile_picture_url}
```

Reply:
![Screenshot from 2024-10-24 11-47-30](https://github.com/user-attachments/assets/aee06c7d-1fd5-42f6-9951-2edca616f493)
removing empty string would cause an error

Reply:
like already said in previous review in another PR, you can update the types to make it accept undefined right? `Avatar` component does accept `undefined`